modules/renter/hostdb/consts.go

```diff
@@ -61,7 +61,7 @@ var (
 	// maxScanSleep is the maximum amount of time that the hostdb will sleep
 	// between performing scans of the hosts.
 	maxScanSleep = build.Select(build.Var{
-		Standard: time.Hour * 4,
+		Standard: time.Hour * 8,
```

Comment:
wait up to 8 hours between scans instead of just 4. Didn't seem to be necessary to be doing 9 scans a day on average. Now it's more like 5.